src/main/java/frc/robot/FieldConstants.java

```diff
@@ -22,4 +25,15 @@ public final class FieldConstants {
     // Center (x) of the Charging Station (in meters)
     public static final double CHARGE_STATION_MIDDLE_X_BLUE = 3.9;
     public static final double CHARGE_STATION_MIDDLE_X_RED = 12.6;
+
+    public static final double FIELD_LENGTH = 16.6;
```

Comment:
We should use the more accurate number from the AprilTag layout file.

Reply:
Is there a field length value in the layout file?